client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -33,15 +33,15 @@ export default React.createClass( {
 
 	render() {
 		const { site, redirect, linkDisplay } = this.props;
-		const buttonElement = linkDisplay ? 'button' : Button;
 
 		const buttonClasses = classNames( {
 			button: true,
 			'disconnect-jetpack-button': true,
 			'is-link': linkDisplay
 		} );
 
-		const buttonProps = assign( {}, this.props, {
+		const omitProps = [ 'site', 'redirect', 'isMock', 'linkDisplay' ];
+		const buttonProps = assign( {}, ...omit( this.props, omitProps ), {
```

Comment:
I'm actually a bit surprised this works as is. I don't think I've seen the object spread operator used this way with the second argument. In any case, two points with the current syntax: (a) you don't need the spread operator and (b) you don't need an empty first object argument, because omit returns a new object and I think it'd be fine to assign into it directly.